src/lava/proc/dense/models.py

```diff
@@ -23,22 +23,23 @@ class PyDenseModelFloat(PyLoihiProcessModel):
     fixed point implementation.
     """
     s_in: PyInPort = LavaPyType(PyInPort.VEC_DENSE, bool, precision=1)
-    a_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE,np.float)
-    a_buff: np.ndarray = LavaPyType(np.ndarray,np.float)
-    weights: np.ndarray = LavaPyType(np.ndarray,np.float)
-    weight_exp: float = LavaPyType(float,np.float)
-    num_weight_bits: float = LavaPyType(float,np.float)
-    sign_mode: float = LavaPyType(float,np.float)
+    a_out: PyOutPort = LavaPyType(PyOutPort.VEC_DENSE, np.float)
+    a_buff: np.ndarray = LavaPyType(np.ndarray, np.float)
+    weights: np.ndarray = LavaPyType(np.ndarray, np.float)
+    weight_exp: float = LavaPyType(float, np.float)
+    num_weight_bits: float = LavaPyType(float, np.float)
+    sign_mode: float = LavaPyType(float, np.float)
 
     def run_spk(self):
         # The a_out sent on a each timestep is a buffered value from dendritic
         # accumulation at timestep t-1. This prevents deadlocking in
         # networks with recurrent connectivity structures.
         self.a_out.send(self.a_buff)
         s_in = self.s_in.recv()
-        self.a_buff = self.weights[:,s_in].sum(axis=1)
+        self.a_buff = self.weights[:, s_in].sum(axis=1)
```

Comment:
self.a_buff[:] is probably faster because it does not require repeated memory allocation.